package.json

```diff
@@ -11,21 +11,20 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "dev": "webpack-dev-server --inline --config ./build/webpack.dev.conf.js",
+    "dev": "vue-cli-service serve",
+    "build": "yarn theo && vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit",
     "start": "npm-run-all --parallel styleguide theo:onchange theo dev",
     "unit": "jest --config ./test/unit/jest.conf.js --coverage",
-    "build": "npm-run-all theo node:build",
-    "build:app": "npm run build",
-    "build:system": "npm-run-all theo node:build:system",
-    "build:docs": "npm-run-all theo styleguide:build",
-    "lint": "eslint '**/*.{js,vue}' --cache",
-    "node:build": "node ./build/build.js",
-    "node:build:system": "node ./build/build-system.js",
-    "styleguide": "vue-styleguidist server --open --config ./config/docs.config.js",
-    "styleguide:build": "vue-styleguidist build --config ./config/docs.config.js",
+    "build:app": "yarn build",
+    "build:system": "yarn build --target lib ./src/system.js",
+    "build:docs": "yarn theo && yarn styleguide:build",
```

Comment:
This will probably also throw an error for some Windows users

Reply:
Why would that be?

Reply:
@elevatebart because of & again :)